styles/Home.module.css

```diff
@@ -52,19 +55,28 @@
   font-size: 2.5rem;
   font-weight: 600;
   max-width: 1000px;
-  color: #086e85;
-  margin: 2rem auto;
+  color: #007ea7;
+  margin: 3rem auto;
 }
 
 .title {
-  font-size: 5rem;
+  font-size: 2rem;
   color: #1d8f89;
```

Comment:
I think this should also be changed to match, like #007ea7

Reply:
After just changing it to match, I think I prefered it green so that there are some more different colors on the site - but it's not that critical so we can see how it looks and update it later on if it ends up feeling weird